### PR TITLE
chore: Update issue templates

### PR DESCRIPTION
**Pull request by elhorton**
first edit, will fix links next

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+*If this is an issue encountered by following a Microsoft document, please open an issue on that page itself. This ensures the fastest targeted response, and you can do this by scrolling all the way down to the bottom of the page to the **Feedback** section to report the issue.*
+
+1. Search existing issues to avoid creating duplicates.
+
+2. If possible test using the [latest release](https://github.com/Azure/azure-iot-sdk-python/releases) to make sure your issues has not already been fixed.
+
+3. Do not share information from your Azure subscription here (connection strings, service names (IoT Hub name, Device Provisioning Service scope ID), etc...). If you need to share any of this information, you can create a ticket and [get assistance from Microsoft Support](https://docs.microsoft.com/en-us/azure/azure-supportability/how-to-create-azure-support-request).
+
+
+4. Include enough information for us to address the bug:
```
Comment (jebrando) [inline .github/ISSUE_TEMPLATE/bug-report.md:26]: Nitpick: Unnecessary blank line

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+5. Delete these instructions before submitting the bug.
+
+Below is a generic bug report format. We recommend you use it as a template and replace the information below each header with your own. 
+
+------------------------------- delete above -------------------------------
+
+
+# Context
+
+- **OS and version used:** <VERSION> (Windows 10, Ubuntu 15.04, etc. )
```
Comment (jebrando) [inline .github/ISSUE_TEMPLATE/bug-report.md:41]: >15.04 [](start = 57, length = 5)

Super Nitpick: Can you please change this to 18.04 the latest Ubuntu LTS

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 1 above)
Comment (elhorton) [reply .github/ISSUE_TEMPLATE/bug-report.md:25]: thanks, fixed

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 2 above)
Comment (elhorton) [reply .github/ISSUE_TEMPLATE/bug-report.md]: good call, updated

**Pull request merged**